pkg/exasol-rest-api/application_runner.go

```diff
@@ -26,7 +27,12 @@ func Run() {
 	swaggerURL := ginSwagger.URL("/swagger/doc.json")
 	AddEndpoints(router, application)
 	router.GET("/swagger/*any", ginSwagger.WrapHandler(swaggerFiles.Handler, swaggerURL))
-	err := router.Run(applicationProperties.ApplicationServer)
+	var err error
+	if applicationProperties.APIUseTLS {
+		err = router.RunTLS(applicationProperties.ApplicationServer, applicationProperties.APITLSCertificatePath, applicationProperties.APITLSPrivateKeyPath)
```

Comment:
Please add an integration test for HTTPS or create a ticket to do this later.

Reply:
See https://github.com/exasol/exasol-rest-api/issues/47